lib/paperclip/storage/google_drive.rb

```diff
@@ -156,7 +156,7 @@ def public_url_for title
       def search_for_title(title)
         parameters = {
                 'folderId' => find_public_folder,
-                'q' => "title contains '#{title}'", # full_title
```

Comment:
I do not know if it is important yet.
I think. all code of gem need refactor and update according changes in GoogleDrive API.

I would be appreciate for some advices.